exercises/two-bucket/example.py

```diff
@@ -0,0 +1,59 @@
+'''
+The solution follows the following approach:
+
+->Pour from first bucket to second
+  1.Fill the first bucket and empty it into the second bucket.
+  2.If first bucket is empty fill it.
+  3.If the second bucket is full empty it.
+  4.Repeat steps 1,2,3 till either the first or second bucket contains the
+    desired amount.
+
+->Pour from first bucket to second
+  1.Fill the second bucket and empty it into the first bucket.
+  2.If second bucket is empty fill it.
+  3.If the firts bucket is full empty it.
+  4.Repeat steps 1,2,3 till either the first or second bucket contains the
+    desired amount.
+'''
+
+
+def two_bucket(bucket_one_cap, bucket_two_cap, desired_liters, first):
+    if first == "one":
+        moves = 0
+        bucket_one, bucket_two = 0, 0
+        while(True):
```

Comment:
Honestly, I don't like these infinity loops. Perhaps we should revisit it later.